model/src/main/java/org/mskcc/cmo/metadb/model/converter/LibrariesStringConverter.java

```diff
@@ -28,6 +28,9 @@ public String toGraphProperty(List<Library> value) {
     public List<Library> toEntityAttribute(String value) {
         List<Library> toReturn = null;
         try {
+            if (mapper.readValue(value, Library[].class) == null) {
+                return null;
+            }
             toReturn = Arrays.asList(mapper.readValue(value, Library[].class));
```

Comment:
we're already catching an exception here so I'm wondering why this additional check is necessary?

Is it possible that the logged stack trace was just making it look like it wasn't being caught and was erroring out instead of returning null?

I want to say I'm mostly certain that we don't need this change

Reply:
we can remove it